node/src/components/small_network/tests.rs

```diff
@@ -350,6 +350,7 @@ async fn run_two_node_network_five_times() {
 /// Sanity check that we can bind to a real network.
 ///
 /// Very unlikely to ever fail on a real machine.
+#[cfg(not(target_os = "macos"))]
```

Comment:
Is this something that needs to be ticketed? Otherwise please add a comment why it doesn't work on MacOS.

Reply:
I believe @xcthulhu was having port binding issues on while developing on osx.